src/panel_state_manager.cpp

```diff
@@ -1095,6 +1095,29 @@ void PanelStateManager::setCEState()
         }
     }
 }
+
+bool PanelStateManager::isFunctionEnabled(const types::FunctionNumber funcNum)
+{
+    auto pos = find_if(panelFunctions.begin(), panelFunctions.end(),
+                       [funcNum](const PanelFunctionality& afunctionality) {
+                           return afunctionality.functionNumber == funcNum;
+                       });
+    if (pos != panelFunctions.end())
+    {
+        return pos->functionActiveState;
+    }
+    return false;
```

Comment:
Just returning flag states only that function is not active, In case function number is not found, some error logging should be there.

Reply:
we discussed this already, if the user given function is not valid, we send the execution status as false.. I think that is enough.

Reply:
returning from DBus is Ok, my point is a log here to know when we return false for Dbus call, was it for some other error or function was disabled.